gnosis/gp-ui
# [Explorer] Add Github + Etherscan logos to external links

# Summary

Closes #1021 

- Added Github logo to Contracts link in footer.
- Added Explorer link to external links in app (excluding tokens).

![Screen Shot 2022-02-09 at 17 58 43](https://user-images.githubusercontent.com/11525018/153291664-7ba81361-8504-4b6d-9b33-eab84baf7f76.png)
![Screen Shot 2022-02-09 at 17 57 53](https://user-images.githubusercontent.com/11525018/153291673-83799594-a544-4972-8cd7-587c7c4d69b0.png)
![Screen Shot 2022-02-09 at 17 57 22](https://user-images.githubusercontent.com/11525018/153291676-1c9deca9-e92a-4b17-8082-9e24e21d5c2a.png)


# To Test

1. Open any page.  i.e:  `Home`
   * You'll see github and explorer logos next to the links in footer. 
2. Open address page. i.e: `/address/0xb6bad41ae76a11d10f7b0e664c5007b908bc77c9`
   * You'll see explorer logo next to the top link. 
3. Open tx page. i.e: `/rinkeby/tx/0x7b6a38411fa9c3cdcd9a9c1fde2d61041300c5567a3d0bbd8755dff29270e5b0`
   * You'll see explorer logo next to the top link.
